appveyor.yml

```diff
@@ -34,14 +36,24 @@ install:
 
 build_script:
   - CD %APPVEYOR_BUILD_FOLDER%
-  - script\build.cmd --code-sign --compress-artifacts
+  - IF [%TASK%]==[installer] (
+      IF [%APPVEYOR_REPO_BRANCH:~-9%]==[-releases] (
+        script\build.cmd --code-sign --compress-artifacts --create-windows-installer
+      ) ELSE (
+        ECHO Skipping installer and Atom build on non-release branch
+      )
+    ) ELSE (
+      ECHO Skipping installer build on non-installer build matrix row &&
```

Comment:
Oh, one more thing... you might want to check out and base it on 1.19-releases. I had to split this block into a new .cmd file to avoid weird line endings with appveyor